odoo/cli/command.py

```diff
@@ -17,54 +20,79 @@ class Command:
 
     def __init_subclass__(cls):
         cls.name = cls.name or cls.__name__.lower()
+        module = cls.__module__.rpartition('.')[2]
+        if not cls.is_valid_name(cls.name):
+            raise ValueError(
+                f"Command name '{cls.name!r}' "
+                f"must match '{COMMAND_NAME_RE.pattern}'")
```

Comment:
Don't enclose your strings in quotes if you are using `!r` :D
```suggestion
                f"Command name {cls.name!r} "
                f"must match {COMMAND_NAME_RE.pattern!r}")
```